param_templates/MOM_input.yaml

```diff
@@ -2717,6 +2717,7 @@ Global:
         datatype: string
         value:
             $OCN_GRID == "MISOMIP": "MISOMIP_IC"
+            else: = "${CASE}.mom6.ic.${RUN_STARTDATE}.nc"
```

Comment:
Is the `.nc` suffix necessary? It looks like it's omitted in both the default value (`MOM_IC`, not `MOM_IC.nc`) and the MISOMIP default (`MISOMIP_IC`, not `MISOMIP_IC.nc`). If it is necessary, we should probably add it to the `MISOMIP` option as well.